src/compiler/checker.ts

```diff
@@ -114,6 +114,9 @@ namespace ts {
             getIdentifierCount: () => sum(host.getSourceFiles(), "identifierCount"),
             getSymbolCount: () => sum(host.getSourceFiles(), "symbolCount") + symbolCount,
             getTypeCount: () => typeCount,
+            getAssignabilityCacheSize: () => assignableRelation.size,

```

Comment:
Thoughts on doing it this way?
```ts
getRelationCacheSizes(): {
   assignable: assignableRelation.size,
   identity: identityRelation.size,
   subtype: subtypeRelation.size
}
```

Reply:
No love for comparablility I see.

Reply:
@RyanCavanaugh seems like a fine idea. do you want to reduce the number of calls when getting perf diagnostics, or reduce the API surface? (or both)

@DanielRosenwasser  lolololololol

Reply:
Yeah, both (seems like you'd rarely want only one of them and it's basically free to query)